doc/nrf/protocols/wifi/index.rst

```diff
@@ -158,3 +162,19 @@ The range of supported rates is vast, ranging from 86 Mbps for a single antenna
 Wi-Fi has traditionally been single user (SU) based, which means that during any particular on-air packet transmission, the communication is between two users (excluding broadcast/multicast scenarios where the same information is delivered to multiple users).
 With the advent of Wi-Fi 6 (and to some extent Wi-Fi 5), multi user (MU) support has been introduced.
 Through both MIMO and Orthogonal Frequency Division Multiple Access (OFDMA) techniques (and even a combination of both), it is now possible to send unique information to multiple users in the same on-air packet transmission, both in the downlink and uplink direction.
+
+.. _ug_wifi_certification:
+
+Wi-Fi certification
+*******************
+
+The Wi-Fi Alliance offers full `certification program <Wi-Fi Certification_>`_ that validates the interoperability of a Wi-Fi end product with other Wi-Fi Certified equipment.
+The program offers three different paths of certification to Contributor-level `members of the Wi-Fi Alliance <Join Wi-Fi Alliance_>`_.
+Implementer-level members can take advantage of the program by implementing Wi-Fi modules that have been previously certified by a Contributor-level member, for example one of the `Nordic third-party modules`_.
+
+From among the three different certfication paths, the following ones are available for end products based on an nRF70 Series device:
+
+* QuickTrack - which lets you apply for certification for a product based on a Wi-Fi Qualified Solution that has only targeted changes to the Wi-Fi functionality, after performing testing at a Wi-Fi test lab.
+* Derivative - which lets you apply for certification without additional testing on the conditions that you are using a copy of a Wi-Fi CERTIFIED device source product and you are not introducing changes to the Wi-Fi functionality.
```

Comment:
I this these lines should be deleted. It it best we let the reader refer to the infocenter document on nRF70 series certification (mentioned in the link below) as the single source of information on the different certification paths